examples/ffd/README.rst

```diff
@@ -2,7 +2,7 @@
 Far-field Voice Local Command
 *****************************
 
-This is the XCORE-VOICE far-field voice local control firmware. Two examples are provided: one example uses the Sensory TrulyHandsfree™ (THF) speech recognition, and the other one uses the Cyberon DSPotter™ speech recognition.
+This is the XCORE-VOICE far-field voice local control firmware. Three examples are provided: all examples include speech recognition and a local dictionary. One example uses the Sensory TrulyHandsfree™ (THF) libraries, and the other ones use the Cyberon DSPotter™ libraries. The two examples with the the Cyberon DSPotter™ libraries differ in the audio source fed into the intent engine. One example uses the audio source from the microphone array, and the other uses the audio source from the I2S interface.
```

Comment:
`I2S` -> `|I2S|`

Reply:
I intentionally avoided using the substitution in this file, as it is not part of any html/pdf documents. It is usually read from github.com, and substitutions are not working there.

@mbanth, let me know if you still prefer using the substitution.

Reply:
You're correct, @lucianomartin, it should not use the substitution.